src/hb-ft.cc

```diff
@@ -273,6 +273,9 @@ hb_ft_font_get_load_flags (hb_font_t *font)
  * Fetches the FT_Face associated with the specified #hb_font_t
  * font object.
  *
+ * This function works with #hb_font_t objects created by
+ * hb_ft_font_create() or hb_ft_font_create_referenced().
+ *
```

Comment:
This probably should be copied to the rest of the hb_ft_font* functions, otherwise it might give the feeling that these two are an exception.

Reply:
Yes, I just noticed that. Will do